src/app/ssmt/header/header.service.ts

```diff
@@ -295,11 +295,14 @@ export class HeaderService {
 
         if (ssmt.boilerInput.blowdownFlashed == false && pressureLevel === 'highPressure') {
           headerType = 'high';
+          let hasHeaderInputs: boolean = ssmt.headerInput.highPressureHeader !== undefined && ssmt.headerInput.highPressure !== undefined && ssmt.headerInput.mediumPressureHeader !== undefined && ssmt.headerInput.mediumPressure !== undefined && ssmt.headerInput.lowPressureHeader !== undefined && ssmt.headerInput.lowPressure !== undefined;
```

Comment:
Don't need to check the mediums. The block that's conditionally wrapped only tries to access high and low